test/e2e/queue.go

```diff
@@ -487,6 +487,32 @@ var _ = Describe("AppWrapper E2E Test", func() {
 
 	})
 
+	It("MCAD appwrapper timeout Test", func() {
+		fmt.Fprintf(os.Stdout, "[e2e] MCAD appwrapper timeout Test - Started.\n")
+		context := initTestContext()
+		var appwrappers []*arbv1.AppWrapper
+		appwrappersPtr := &appwrappers
+		defer cleanupTestObjectsPtr(context, appwrappersPtr)
+
+		aw := createGenericAWTimeoutWithStatus(context, "aw-test-jobtimeout-with-comp-1")
+		err1 := waitAWPodsReady(context, aw)
+		Expect(err1).NotTo(HaveOccurred())
+		time.Sleep(1 * time.Minute)
```

Comment:
Should we make this `time.Sleep(60 * time.Second)` just to be consistent and do "everything" in seconds?